libs/core/langchain_core/prompts/prompt.py

```diff
@@ -213,13 +213,15 @@ def from_examples(
     def from_file(
         cls,
         template_file: Union[str, Path],
+        encoding: Optional[str] = "utf-8",
```

Comment:
This is a breaking change for users that have been passing the encoding by position. You can move it after input_variables and use the `*` operator to force invocation by name.

Change looks good, could you add a simple unit test given that this is langchain-core?